### DISCUSSION TITLE
PaddleX/latest/pipeline_usage/tutorials/ocr_pipelines/OCR

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleX#2605] [General]
# PaddleX/latest/pipeline_usage/tutorials/ocr_pipelines/OCR

https://paddlepaddle.github.io/PaddleX/latest/pipeline_usage/tutorials/ocr_pipelines/OCR.html

**Comment by ming2681** [comment]
在调用det的时候没有问题，在调用rec的时候，总是提示File "C:\ProgramData\miniconda3\envs\paddlex3\lib\site-packages\paddlex-3.0.0b0-py3.10.egg\paddlex\modules\text_recognition\predictor\predictor.py", line 38, in load_other_src
    raise FileNotFoundError(
FileNotFoundError: Cannot find config file: D:\AI\PaddleX3\output\ocr\PP-OCRv4\ch_PP-OCRv4_rec_infer\inference.yml

我在对应的路径已经放置了rec模型

**Reply by cuicheng01** [reply]
这个rec模型是从哪里得到的呢？请使用paddlex的模型，paddlx对模型做了统一的规范，新增了inference.yml，用来管理预处理、后处理、字典等
